Sources/Protocols/Keyboardable.swift

```diff
@@ -1,53 +0,0 @@
-//
```

Comment:
Are you sure about the removal of the Keyboardable? I wonder if we don't want to keep it for backward compatibility with older projects or any use cases where height constraint would not be comfortable to use... What do you think?

Reply:
I think we have only one internal app, that uses `Keyboardable` since 1.0 release. And the solution to migration is really simple by just copying the protocol to the app. So, I think this is not so critical change.

Reply:
Ok, we can always put it back if there's a need for that.